src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs

```diff
@@ -99,7 +99,8 @@ IConfiguration configuration
                 if (pullRequestModel == null || pullRequestModel.Commits.Any(c=> c== commitSha))
                 {
                     // PR commit is already processed. No need to reprocess it again.
-                    return "";
+                    return pullRequestModel != null && !string.IsNullOrEmpty(pullRequestModel.ReviewId)?

```

Comment:
Would it make sense to separate the validation for `if (pullRequestModel == null) return "";` ? I can see how that will help reading the flow of the code

Reply:
changed